### ISSUE TITLE
Using same reference for "nothing" sprite

### ISSUE THREAD

**Issue by Atria1234** [notnotmelon/fluid-memory-storage#2]
Just to finish the conversation

You are right, the sprite really is there and problem is with how `serpent` shows it.

But issue also seems to be in my mod which modifies the "nothing" table. The problem in my case is that all those properties don't have same type and when my mod tries to modify `gas_flow` it processes it like it was an animation of one frame. But when it processes it, it no longer is valid `Sprite` type which is needed for other properties which still refenrece same table.

Bottom line is that I could also handle this by making copy of every table I process. So thanks for fixing this in your mod, saved me a job ❤️
